.github/actions/spelling/expect.txt

```diff
@@ -11,6 +11,7 @@ certificatechains
 cmek
 Cooldown
 cryptokey
+cpes
```

Comment:
We should stick this in the main dictionary. It's going to show up all over.

Reply:
agreed, would love to do this as a separate task

Reply:
It's in the main dictionary now

Reply:
Merci, I am going to remove it from here then